install/ui/src/freeipa/topology.js

```diff
@@ -507,6 +507,9 @@ topology.servers_search_facet = function(spec, no_init) {
                 }
             }
 
+            // Do not show warning if there is only one master
+            if (result.length <= 1) return;
```

Comment:
I'd expect this check to appear prior to the `for` cycle so no useless computations are done, other than that LGTM. I will test it when I'm fixing https://github.com/freeipa/freeipa/pull/553.